### ISSUE TITLE
Pleasing theme

### ISSUE THREAD

**Issue by Moirvae** [cocaseattle/artist-directory#24]
We need to have the site look somewhat similar to the existing one, so that people know they're in the right place, somewhat like the current main site (ditto), but updated to distinguish it from both. Some sort of branding/logo/color/font coordination. 

I will look for a few likely themes and install them, to try them out.

**Comment by Moirvae** [comment]
Trying out Koji, a minimal theme, and a new static home page.

**Comment by HTD-CoCA** [comment]
Adding Candy and I.  We need to agree on a theme  that would not only look nice but also not conflicting with our plugins.

**Comment by HTD-CoCA** [comment]
Added and activated the theme called "Freeware."

**Comment by HTD-CoCA** [comment]
This theme is working with all of our current plugins and can be modified if needed.

**Comment by HTD-CoCA** [comment]
Close issue.

**Comment by Moirvae** [comment]
I agree that the theme needs to work with all plugins; that is a separate issue of "Functional theme" which I agree has been met. However, the original point, of a theme that works for our artist members, needs to stay open while we finish out the details of the site. This can remain on my plate, along with IA, as they are intertwined.
